### PR TITLE
Reorganization of demo steps

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Created additional step (having single step is a shame)
Changed step(s) to be available as part of different organization. 
This organization can encompass some sample demo steps that will help sales team to show how Raincatcher may work. 

Note: I couldn't test that due to some issues i encountered in my local env.
Changes in core will be required to fully verify it. Posting just for early feedback.

I think that we will need to invest a little bit more into step creation (maybe sample step creation guide) as there is no way anyone will be able to create step themselves without that.
This work may be blocked by step framework refactoring.

ping @austincunningham

### CODE REVIEW

File: packages/step-accident-form/lib/angular/template/accident.tpl.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<md-subheader>
+  Vehicle Inspection</md-subheader>
```
Comment (paolobueno) [inline packages/step-accident-form/lib/angular/template/accident.tpl.html]: Missing header update

File: packages/step-vehicle-inspection/LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```
Comment (paolobueno) [inline packages/step-vehicle-inspection/LICENSE]: Should we use MIT for everything? I thought we were on apache2

File: packages/step-vehicle-inspection/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@raincatcher/vehicle-inspection",
+  "name": "@raincatcher-demo/demo-step-vehicle-inspection",
```
Comment (paolobueno) [inline packages/step-vehicle-inspection/package.json]: is this the right name here? looks like it's referenced by `@raincatcher-demo/vehicle-inspection` in other places

File: packages/step-vehicle-inspection/LICENSE (hunk 2 above)
Comment (wtrocki) [reply packages/step-vehicle-inspection/LICENSE]: We should use apache2. None of the modules will need license file. Going to remove it.
License is in package.json

Comment (wtrocki) [reply packages/step-vehicle-inspection/LICENSE]: Btw. BSD license for demo apps makes sense.

File: packages/workorder-angular/package.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/feedhenry-raincatcher/raincatcher-angularjs.git"
   },
   "author": "feedhenry-raincatcher@redhat.com",
-  "license": "MIT",
+  "license": "Apache 2.0",
```
Comment (paolobueno) [inline packages/workorder-angular/package.json]: I think the right string is `Apache-2.0`, see https://spdx.org/licenses/Apache-2.0.html

With this one I get `npm WARN @raincatcher/angularjs-workorder@1.0.0 license should be a valid SPDX license expression`

**Review by paolobueno — DISMISSED** [review]

**Review by JameelB — APPROVED** [review]

**Pull request merged**